google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/datacatalog_facade.py

```diff
@@ -106,7 +110,18 @@ def upsert_entry(self, entry_group_name, entry_id, entry):
         return persisted_entry
 
     @classmethod
-    def __entry_was_updated(cls, current_entry, new_entry):
+    def __entry_was_updated(cls, *, current_entry, new_entry):
+        """
+        Compares if the current_entry was updated.
+        Since the args order must be respected, this method uses
+        named args.
```

Comment:
```suggestion
        keyword-only arguments.
```

Reply:
Removed the key-args, this is not longer needed.